cdmtaskservice/nersc/manager.py

```diff
@@ -38,7 +36,6 @@
 _MANIFEST_ROOT_DIR = Path("cdm_task_service")
 
 # TODO PROD add start and end time to task output and record
-# TODO TICKET ask NERSC for start & completion times for tasks
```

Comment:
Kind of pointless as task records are no longer visible in the API 10 minutes after completion